templates/errors/500.html

```diff
@@ -6,12 +6,12 @@
   <h1>{{ _("Sorry, there is a problem with this service") }}</h1>
   <p>{{ _("Try again later.") }}</p>
   <p>{{ _("If you have started a survey, your answers have been saved.") }}</p>
-  {% if survey_type and (survey_type in SURVEY_TYPES_BUSINESS or survey_type in SURVEY_TYPES_DEFAULT or survey_type in SURVEY_TYPES_SOCIAL) %}
+  {% if survey_type and (survey_type in SURVEY_TYPES_BUSINESS or survey_type in SURVEY_TYPES_DEFAULT or survey_type in SURVEY_TYPES_SOCIAL or survey_type in SURVEY_TYPES_HEALTH) %}
```

Comment:
Minor - could create a var for all survey types rather than chaining these or's?